model-mesh/default/config-defaults.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# These are the system defaults which users can override with a user config
+# Settings here will be overridden by future product releases and should not
+# be changed unless directed by IBM support
+podsPerRuntime: 2
+headlessService: true
+modelMeshImage:
+  name: kserve/modelmesh
```

Comment:
I think we should move these images over to the opendatahub Quay org and refer to them. Dockerhub images will run into the rate-limiting stuff

Reply:
I agree...I've mirrored and updated.